unicodetools/org/unicode/propstest/CheckEmojiProps2.java

```diff
@@ -0,0 +1,30 @@
+package org.unicode.propstest;
+
+import org.unicode.props.IndexUnicodeProperties;
+import org.unicode.props.UcdProperty;
+import org.unicode.props.UcdPropertyValues.Binary;
+
+import com.google.common.collect.Sets;
+import com.ibm.icu.text.UnicodeSet;
+
+/**
+ * PR #44 added this class, but it was obviously added under the same name with
+ * different code at another time. Re-adding under a different name.
+ */
```

Comment:
```suggestion
/*
Note:
- in Feb 2021  org.unicode.propstest.CheckEmojiProps was added in 968c94c (PR#44) but not merged.  To reduce confusion, this class has been renamed to CheckEmojiProps2.
- in June 2021 added org.unicode.propstest.CheckEmojiProps was added in 163def1 in CLDR 14.0 keyword fixes (PR#65) with different content.
- Both of these are not to be confused with org.unicode.tools.CheckEmojiProps 
 */
```

@markusicu sound better?

Reply:
suggested change lg